rel-eng/openshift-node.service

```diff
@@ -4,7 +4,7 @@ After=docker.service
 After=openshift-master.service
 After=openshift-sdn-master.service
 After=openshift-sdn-node.service
-Requires=docker.service
+Wants=docker.service
```

Comment:
Why this change? Is it to enable to run with non-systemd docker? Or a mistake...